apps/site-landing/src/containers/ReturnProfile/data/state.ts

```diff
@@ -73,6 +73,10 @@ export const increment = (state: SimulationState, date: DateTime) => ({
 export const toFiat = (coin: Decimal, data: MarketData) => coin.mul(data.P)
 export const toCoin = (coin: Decimal, data: MarketData) => coin.div(data.P)
 
-export const calcFiat = (state: SimulationState, data: MarketData[]) =>
+export const grossFiat = (state: SimulationState, data: MarketData[]) =>
```

Comment:
gross in finance means raw number without costs (in the sim, the credit being carried is the cost)